.travis.yml

```diff
@@ -14,53 +15,89 @@ env:
     - BASEPATH: "$TRAVIS_BUILD_DIR"
     - ROOTSYS: "$BASEPATH/root"
 
+addons:
+  homebrew:
+    packages:
+      - boost
+      - python3
+      - pip3
+  apt:
+    update: true
+    packages:
+      - libboost-all-dev
+      - python3
+      - python3-pip
+      - python3-venv
+      - pandoc
```

Comment:
I like the version before better. If you can make it to work that there is some base requirements being installed for every instance and then each instance adds some specials dep on top, this would be the way to go. Otherwise I would go back to the previous.

Reply:
Apart from `pandoc` jobs require these addons, so I don't see the problem

Reply:
no libboost is also different from what I saw

Reply:
You are trading writing a few lines of code with runtime, I dont see the benefit in this

Reply:
`libboost` was in the jobs before, only not in the pip package deploy (why does that one have `addons` actually).

> You are trading writing a few lines of code with runtime, I dont see the benefit in this
The runtime would only be for pandoc. If this has a critical effect on runtime, I can remove pandoc in the root, and overwite the addons for the documentation job.

Reply:
all right, thats fine

Reply:
Still, I did wonder about the addons for the pip deploy job. It [used to be](https://github.com/ComPWA/pycompwa/blob/911ea165ac5b4c7efecafab7e49273abd5346aef/.travis.yml#L107) `python3`, `python3-pip`, and `python3-venv`, but what was that needed for?

Reply:
Im not sure why die pip package does not include that. Looks like a bug to me (and since now deployment has been done for a while it was not noticed). So that would be an advantage of your setup.

Reply:
I also tried the [conventional travis setup for python](https://docs.travis-ci.com/user/languages/python/), but that became a mess with building ComPWA